ktor-server/ktor-server-plugins/ktor-server-openapi/jvm/src/io/ktor/server/plugins/openapi/OpenAPI.kt

```diff
@@ -49,13 +50,25 @@ public fun Routing.openAPI(
     }
 }
 
-internal fun Routing.resolveOpenAPIFile(swaggerFile: String): File {
-    val resource = application.environment.classLoader.getResource(swaggerFile)
-    val file = if (resource != null) File(resource.toURI()) else File(swaggerFile)
+internal fun Route.resolveOpenAPIFileContent(swaggerFile: String): String {
```

Comment:
The extension of the `Route` class is misleading because this method only reads a resource or a file and has nothing to do with routing. This class is only needed to get a class loader.

Reply:
I'm not sure that resolving means looking at different locations and reading.

Reply:
ok, let me name it `readOpenAPIFile` instead